cylc/flow/prerequisite.py

```diff
@@ -47,26 +46,26 @@
     from cylc.flow.id import Tokens
 
 
-AnyPrereqMessage = Tuple[Union['PointBase', str, int], str, str]
+AnyPrereqTuple = Tuple[Union['PointBase', str, int], str, str]
 
 
-class PrereqMessage(NamedTuple):
-    """A message pertaining to a Prerequisite."""
+class PrereqTuple(NamedTuple):
+    """A condition in a Prerequisite."""
```

Comment:
```suggestion
    """An output in a Prerequisite."""
```

(`a & b` is a condition)